boards/arm/efm32hg_slstk3400a/doc/efm32hg_slstk3400a.rst

```diff
@@ -0,0 +1,176 @@
+.. _efm32hg_slstk3400a:
+
+EFM32HG-SLSTK3400A
+##################
+
+Overview
+********
+
+The EFM32 Happy Gecko Starter Kit EFM32HG-SLSTK3400A contains a MCU from the
+EFM32HG family built on ARM® Cortex®-M0+ processor with excellent low
+power capabilities.
+
+.. figure:: happy-gecko-starter-kit.jpg
+   :width: 375px
+   :align: center
+   :alt: EFM32HG-SLSTK3400A
+
+   EFM32HG-SLSTK3400A (image courtesy of Silicon Labs)
+
+Hardware
+********
+
+- Advanced Energy Monitoring system for precise current tracking
+- Real-time energy and power profiling
+- ARM Cortex M0+ with 64 kB Flash and 8 kB RAM
+- 128 X 128 pixel Memory LCD
+- 2 user buttons, 2 user LEDs and 2 touch buttons
+- 20 pin expansion header
+- Silicon Labs Si7021 Relative Humidity/Temperature sensor
+- USB device interface
+- Integrated SEGGER J-Link USB debugger/emulator with debug out functionality
+
+
+See these documents for more information
+
+- `EFM32HG Website`_
+- `EFM32HG Datasheet`_
+- `EFM32HG Reference Manual`_
+- `EFM32HG-SLSTK3400A Website`_
+- `EFM32HG-SLSTK3400A User Guide`_
+- `EFM32HG-SLSTK3400A Schematics`_
+
+Supported Features
+==================
+
+The efm32hg_slstk3400 board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
```

Comment:
delete the blank line and indent:

    The default configuration can be found in the defconfig file:
    ``boards/arm/efm32hg_slstk3400a/efm32hg_slstk3400a_defconfig``